### ISSUE TITLE
Array to string conversion

### ISSUE THREAD

**Issue by mu-hasan** [knuckleswtf/scribe#97]
- [x] **I confirm that I have read and attempted the tips in the [Troubleshooting Guide](https://scribe.readthedocs.io/en/latest/troubleshooting.html).**

**What happened?**
1. I set my configuration to config/scribe.php
2. Then I ran `php artisan scribe:generate`
3. But I saw
```
Array to string conversion (View: /usr/share/nginx/api/core/vendor/knuckleswtf/scribe/resources/views/partials/example-requests/php.blade.php)
```
instead of...

**Screenshots and stack traces:**
```
at vendor/knuckleswtf/scribe/src/../resources/views//partials/example-requests/php.blade.php:20
    16| @foreach($route['cleanBodyParameters'] as $parameter => $value)
    17| @foreach(\Knuckles\Scribe\Tools\WritingUtils::getParameterNamesAndValuesForFormData($parameter,$value) as $key => $actualValue)
    18|             [
    19|                 'name' => '{!! $key !!}',
  > 20|                 'contents' => '{!! $actualValue !!}'
    21|             ],
    22| @endforeach
    23| @endforeach
    24| @foreach($route['fileParameters'] as $parameter => $file)
```

**My environment:**
 - PHP version: 7.4.7
 - Framework: Laravel
 - Laravel version: 7.19.0
 - Scribe version: 1.4.1

**My Scribe config (minus the comments):**
```
<?php

return [
    'type' => 'static',
    'static' => [
        'output_path' => 'public/docs',
    ],
    'laravel' => [
        'add_routes' => true,
        'docs_url' => '/docs',
        'middleware' => [],
    ],
    'auth' => [
        'enabled' => false,
        'in' => 'bearer',
        'name' => 'token',
        'use_value' => env('SCRIBE_AUTH_KEY'),
        'extra_info' => 'You can retrieve your token by visiting your dashboard and clicking <b>Generate API token</b>.',
    ],
    'intro_text' => "Welcome to our API documentation!
        <aside>As you scroll, you'll see code examples for working with the API in different programming languages in the dark area to the right (or as part of the content on mobile), and you can switch the programming language of the examples with the tabs in the top right (or from the nav menu at the top left on mobile).</aside>",
    'example_languages' => [
        'javascript',
        'php',
        'bash',
    ],
    'base_url' => null,
    'title' => null,
    'description' => '',
    'postman' => [
        'enabled' => true,
        'base_url' => null,
        'description' => null,
        'auth' => null,
        'overrides' => [],
    ],
    'openapi' => [
        'enabled' => false,
        'overrides' => [],
    ],
    'default_group' => 'Unspecified',
    'logo' => '../img/logo-docs.png',
    'router' => 'laravel',
    'routes' => [
        [
            'match' => [
                'domains' => ['*'],
                'prefixes' => ['api/*'],
                'versions' => ['v1'],
            ],
            'include' => [],
            'exclude' => [
                'api/pay-reregistration',
                'api/generate/location',
                'api/winpay-listener',
            ],
            'apply' => [
                'headers' => [
                    'Content-Type' => 'application/json',
                    'Accept' => 'application/json',
                ],
                'response_calls' => [
                    'methods' => ['GET'],
                    'config' => [],
                    'cookies' => [],
                    'queryParams' => [],
                    'bodyParams' => [],
                    'fileParams' => [],
                ],
            ],
        ],
    ],

    'fractal' => [
        'serializer' => null,
    ],

    'faker_seed' => null,

    'strategies' => [
        'metadata' => [
            \Knuckles\Scribe\Extracting\Strategies\Metadata\GetFromDocBlocks::class,
        ],
        'urlParameters' => [
            \Knuckles\Scribe\Extracting\Strategies\UrlParameters\GetFromUrlParamTag::class,
        ],
        'queryParameters' => [
            \Knuckles\Scribe\Extracting\Strategies\QueryParameters\GetFromQueryParamTag::class,
        ],
        'headers' => [
            \Knuckles\Scribe\Extracting\Strategies\Headers\GetFromRouteRules::class,
            \Knuckles\Scribe\Extracting\Strategies\Headers\GetFromHeaderTag::class,
        ],
        'bodyParameters' => [
            \Knuckles\Scribe\Extracting\Strategies\BodyParameters\GetFromFormRequest::class,
            \Knuckles\Scribe\Extracting\Strategies\BodyParameters\GetFromBodyParamTag::class,
        ],
        'responses' => [
            \Knuckles\Scribe\Extracting\Strategies\Responses\UseTransformerTags::class,
            \Knuckles\Scribe\Extracting\Strategies\Responses\UseResponseTag::class,
            \Knuckles\Scribe\Extracting\Strategies\Responses\UseResponseFileTag::class,
            \Knuckles\Scribe\Extracting\Strategies\Responses\UseApiResourceTags::class,
            \Knuckles\Scribe\Extracting\Strategies\Responses\ResponseCalls::class,
        ],
        'responseFields' => [
            \Knuckles\Scribe\Extracting\Strategies\ResponseFields\GetFromResponseFieldTag::class,
        ],
    ],

    'routeMatcher' => \Knuckles\Scribe\Matching\RouteMatcher::class,

    'continue_without_database_transactions' => [],
];
```
**Additional info:**

**Comment by shalvah** [comment]
> I confirm that I have read and attempted the tips in the Troubleshooting Guide.

Did you really? Because one of the tips says to update your installation. Yet you're reporting 1.4.1 when the latest release is 1.7.0.

**Comment by mu-hasan** [comment]
I just have updated to 1.7.0 and still error

**Comment by mu-hasan** [comment]
I know how to produce the bug

```
    /**
     * Create Exam Question
     * 
     * Endpoint untuk buat baru pertanyaan berdasarkan paket soal ujian
     *
     * @bodyParam question string required Isian Pertanyaan. Example: Bahasa Indonesia Kelas X-A
     * @bodyParam image file optional Diisi jika ingin menggunakan gambar.
     * @bodyParam choices array optional Diisi jika ingin multiple choice.
     * @bodyParam choices.* object optional Item multiple choice.
     * @bodyParam choices.*.answer string optional Caption Jawaban. Diisi jika ingin multiple choice. Example: Matahari Terbenam
     * @bodyParam choices.*.is_true boolean optional Status jawaban benar/salah. Diisi jika ingin multiple choice. Example: true
     * 
     * @resource App\Http\Resources\School\ExamQuestionResource
     * @resourceModel App\ExamQuestion
     */
```

**Comment by shalvah** [comment]
Thanks for sharing an example. That's really helpful. My guess would be that this is caused by the array of objects you have not playing nice with the PHP example template.

However, this is a bug I won't be fixing. v2 (which is currently in the works) introduces a new syntax for array and object parameters (nothing complex, but not based on Laravel's validation syntax). V2 should be available at least in beta sometime within the next two weeks.

Also, in the future, make sure to actually try the troubleshooting guidelines, not just say that you've done so.

**Comment by shalvah** [comment]
Fixed (v1.8.1)
